backend/internal/server/apis/sports/stats/goals_test.go

```diff
@@ -0,0 +1,50 @@
+package stats
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/models"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPostGoalBadRequest(t *testing.T) {
+	app := apis.CreateTestApp()
+
+	req, _ := http.NewRequest(fiber.MethodPost, "/api/v1/goals", nil)
```

Comment:
The file `/api/v1/goals` has been moved. This could be the problem for the failure.